docs/image.rst

```diff
@@ -39,8 +51,17 @@ Classes
 
     The other form creates an empty image with ``width`` columns and
     ``height`` rows. Optionally ``buffer`` can be an array of
-    ``width``×``height`` integers in range 0-9 to initialize the image.
-
+    ``width``×``height`` integers in range 0-9 to initialize the image:
```

Comment:
Adding the extra colon to format the code line bellow in the same format as the previous code examples.
```suggestion
    ``width``×``height`` integers in range 0-9 to initialize the image::
```